components/BurgerMenuAndSwitch.js

```diff
@@ -0,0 +1,109 @@
+import {
+  useColorMode,
+  Switch,
+  Flex,
+  Button,
+  IconButton,
+} from "@chakra-ui/react";
+import { useState } from "react";
+import { HamburgerIcon, CloseIcon } from "@chakra-ui/icons";
+import NextLink from "next/link";
+
+export const BurgerMenu = () => {
+  {
+    /*const { colorMode, toggleColorMode } = useColorMode();
+const isDark = colorMode === "dark";
+useless switchmode, just testing*/
+  }
+
+  const [changeDisplay, setChangeDisplay] = useState("none");
```

Comment:
It works to keep a string in state here. But it does not really communicate your intent and is coupled to the css implementation. Please consider using a boolean state alá:
```suggestion
  const [isOpen, setIsOpen] = useState(false);
```

Reply:
That was my first impression as well, but later it is changed to `Flex`. The `none` makes it hard not go the `true/false` route.
Like I mentioned in my review, the state might not necessary, because `changeDisplay` is not used anyway.